### PR TITLE
Creating initial db values

### PR DESCRIPTION
**Pull request by vakhonin**

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 1]
```diff
@@ -17,51 +18,52 @@
...
 import static com.epam.magazinestore.util.MessageUtil.UNSUCCESSFUL_REGISTRATION;
 import static com.epam.magazinestore.util.PathUtils.ACCOUNT;
 import static com.epam.magazinestore.util.PathUtils.CREATE_ACCOUNT;
 import static com.epam.magazinestore.util.PathUtils.LOGIN;
-import static com.epam.magazinestore.util.VariableNameUtil.ACCOUNT_NAME_VARIABLE;
-import static com.epam.magazinestore.util.VariableNameUtil.ACCOUNT_PASSWORD_VARIABLE;
-import static com.epam.magazinestore.util.VariableNameUtil.MESSAGE_JSP_VARIABLE;
-import static com.epam.magazinestore.util.VariableNameUtil.NAME_JSP_VARIABLE;
-import static com.epam.magazinestore.util.VariableNameUtil.SESSION_ACCOUNT_VARIABLE;
+import static com.epam.magazinestore.util.VariableNameUtil.MESSAGE;
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: VariableNameUtil is unnecessary, strings and constants which are used in one class file you need to create in the same class file.

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 2]
```diff
@@ -17,51 +18,52 @@
...
     @PostMapping(value = ACCOUNT)
-    public ModelAndView account(@RequestParam(ACCOUNT_NAME_VARIABLE) String name, @RequestParam(ACCOUNT_PASSWORD_VARIABLE) String password, ModelAndView modelAndView) {
+    public ModelAndView account(@RequestParam("name") String name, @RequestParam("password") String password, ModelAndView modelAndView) {
 
         if(accountService.isUserNameExists(name)) {
             Account account = accountService.getAccountByUsername(name);
             if(account.getPassword().equals(password)) {
-                modelAndView.addObject(NAME_JSP_VARIABLE, account.getUsername());
+                modelAndView.addObject(NAME, account.getUsername());
                 Account acc = new Account();
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: What about Builder for Account ?

File: src/main/java/com/epam/magazinestore/controller/IndexController.java [hunk 3]
```diff
@@ -34,10 +34,10 @@ public String out(SessionStatus sessionStatus) {
     }
 
     @GetMapping(value = HOME)
-    public ModelAndView home(ModelAndView modelAndView, @ModelAttribute(SESSION_ACCOUNT_VARIABLE) Account acc) {
+    public ModelAndView home(ModelAndView modelAndView, @ModelAttribute(SESSION_ACCOUNT) Account acc) {
 
         if(acc.getUsername() != null) {
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/IndexController.java]: Objects.nonNull

File: src/main/java/com/epam/magazinestore/util/VariableNameUtil.java [hunk 4]
```diff
@@ -1,9 +1,7 @@
 package com.epam.magazinestore.util;
 
 public class VariableNameUtil {
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/util/VariableNameUtil.java]: Unnecessary files

File: src/main/java/com/epam/magazinestore/controller/AccountController.java (hunk 1 above)
Comment (vakhonin) [reply src/main/java/com/epam/magazinestore/controller/AccountController.java]: Why is it unnecessary?? For example, NAME and SESSION_ACCOUNT are used in 2 classes.

Comment (AntonKiselyov) [reply src/main/java/com/epam/magazinestore/controller/AccountController.java]: Using of path-, messagesutils.class allows you to save urls and messages , configure and to change only strings
you can define urls and messages not only in class but also in properties and another file.
JspTitles can used in different controllers, the amount of controllers can be more than 2 

Here I say about constants which can be used in only one class, Yes, you can find them not in one class, but this is the best practice to define own constants in the same class.

**Pull request merged**